### DISCUSSION TITLE
Install over IOS

### DISCUSSION THREAD

**Discussion by robertopuentesdiaz** [capawesome-team/capacitor-firebase#393] [Q&A] [answered]
Hi there. I try to use 
    "@capacitor-firebase/performance": "1.3.0",
on my Ionic Project... But I don't find Docs for IOS.
I get it to work over Android. What do I need to do?

**Comment by robingenz** [comment, marked as answer]
Here you can find the documentation for version `1.3.0`: https://github.com/capawesome-team/capacitor-firebase/tree/%40capacitor-firebase/authentication%401.3.0/packages/authentication#installation

Please note that I can only provide support for the latest version.

**Reply by robertopuentesdiaz** [reply]
Thanks for response. i try use **performance**
I still use Capacitor 4, so i cann't migrate yet. Then` npx cap sync`  return :

```

[error] Analyzing dependencies
        [!] CocoaPods could not find compatible versions for pod "Firebase/Performance":
        In Podfile:
        CapacitorFirebasePerformance (from `../../node_modules/@capacitor-firebase/performance`) was resolved to 1.4.0,
        which depends on
        Firebase/Performance (= 10.2.0)
        
        None of your spec sources contain a spec satisfying the dependency: `Firebase/Performance (= 10.2.0)`.
        
        You have either:
        * out-of-date source repos which you can update with `pod repo update` or with `pod install --repo-update`.
        * mistyped the name or version.
        * not added the source repo that hosts the Podspec to your Podfile.
```

**Reply by robingenz** [reply]
There it says what you should do: Go to the iOS folder and run `pod install --repo-update`:

```bash
cd ios/app && pod install --repo-update
```
